pages/service.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <title>Service</title>
+</head>
+<body>
+</head>
+<div class="back-drop"></div>
+    <header>
+        <div class="navigation-background__color">
+            <div class="navigation-container">
+                <div class="navigation-logo">
+                    <a href="#">
+                        <img src="../images/logo-light.png" height="60" alt="Inspire Academy logo white">
+                    </a>
+                </div>
+                <nav>
+                    <div class="hamburger-container">
+                        <div class="hamburger-button"><div class="before-after"></div></div>
+                    </div>
+                    <ul>
+                        <li><a href="#">INSPIRE BOOKS</a></li>
+                        <li><a href="#">22 CENTURY EDUCATION</a></li>
+                        <li><a href="#">TUTORING</a></li>
+                        <li><a href="#">SERVICES</a></li>
+                        <li><a href="#">ABOUT US</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+    </header>
+    <!--mobile navigation bar--> 
+    <div class="mobile-navigation">
+        <nav>
+                <ul>
+                    <li><a href="#">INSPIRE BOOKS</a></li>
+                    <li><a href="#">22 CENTURY EDUCATION</a></li>
+                    <li><a href="#">TUTORING</a></li>
+                    <li><a href="#">SERVICES</a></li>
+                    <li><a href="#">ABOUT US</a></li>
+                </ul>
+            </nav>
+    </div>
+    <div>
```

Comment:
please wrap this <div> inside the <main> tag also

Reply:
okay!